tests/test_image_featurizer.py

```diff
@@ -235,10 +235,10 @@ def test_load_data_multiple_columns(featurizer_autosample):
     compare_featurizer_class(featurizer_autosample, (227, 227), np.zeros((1)), **COMPARE_ARGS_MULT)
 
 
-def test_load_and_featurize_save_csv_and_features(featurizer):
+def test_featurize_save_csv_and_features(featurizer):
```

Comment:
All of the following changes are just minor renamings of the test functions to match with the actual name of the methods being tested